frontend/src/components/previewers/Audio/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+	"name": "basic-audio-component",
+	"version": "1.0.0",
+	"description": "A React component for audio",
+	"main": "Audio.tsx",
+	"dependencies": {
+		"clowder2-core": "1.0.0",
+		"react": "^17.0.2",
+		"react-dom": "^17.0.2"
+	},
+	"vizConfig": {
+		"name": "Audio",
+		"mainType": "audio",
+		"mimeType": [
```

Comment:
Since this is a list, would using the plural form (`mimeTypes`) maybe better? What do you think?